CONTRIBUTING.md

```diff
@@ -50,50 +94,22 @@ remote client info, request content (with value field redacted), request
 handling latency, response size, etc. Uses zap logger if available,
 otherwise uses capnslog.
 
-Fixes #38
+Signed-off-by: FirstName LastName <github@github.com>
 ```
 
-The format can be described more formally as follows:
+## Create a pull request
 
-```
-<package>: <what changed>
-<BLANK LINE>
-<why this change was made>
-<BLANK LINE>
-<footer>
-```
+Please follow [making a pull request](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request) guide.
 
-The first line is the subject and should be no longer than 70 characters, the second
-line is always blank, and other lines should be wrapped at 80 characters. This allows
-the message to be easier to read on GitHub as well as in various git tools.
+If you are still working on the pull request, you can convert it to draft by clicking `Convert to draft` link just below list of reviewers.
 
-### Pull request across multiple files and packages
+## Get your pull request reviewed
```

Comment:
We should clearly state that small PR is better. If it's a big PR (e.g. > 500 lines of change), then suggest the contributors to breakdown the PR as possible as they can.